docs/advanced/addons/kubeovn-operator.md

```diff
@@ -0,0 +1,209 @@
+    lsCtSkipOstLportIPS: true
+    lsDnatModDlDst: true
+    secureServing: false
+    setVLANTxOff:  false
+    u2oInterconnection:  false
+  debug:
+    mirrorInterface: mirror0
+  dpdkCPU:  "0"
+  dpdkMEMORY: "0"
+  dpdkVersion: "19.11"
+  dualStack:
+    joinCIDR: fd00:100:64::/112
+    pingerExternalAddress: 2606:4700:4700::1111
+    pingerExternalDomain: google.com.
+    podCIDR: fd00:10:16::/112
+    podGateway: fd00:10:16::1
+    serviceCIDR: fd00:10:96::/112
+  global:
+    images:
+      kubeovn:
+        dpdkRepository: kube-ovn-dpdk
+        repository: kube-ovn
+        supportArm: true
+        thirdParty:  true
+        vpcRepository: vpc-nat-gateway
+    registry:
+      address: docker.io/kubeovn
+  hugePages: "0"
+  hugepageSizeType:  hugepages-2Mi
+  imagePullPolicy: IfNotPresent
+  ipv4:
+    joinCIDR: 100.64.0.0/16
+    pingerExternalAddress: 1.1.1.1
+    pingerExternalDomain: google.com.
+    podCIDR: 10.52.0.0/16
+    podGateway: 10.52.0.1
+    serviceCIDR: 10.52.0.1
+  ipv6:
+    joinCIDR: fd00:100:64::/112
+    pingerExternalAddress: 2606:4700:4700::1111
+    pingerExternalDomain: google.com.
+    podCIDR: fd00:10:16::/112
+    podGateway: fd00:10:16::1
+    serviceCIDR: fd00:10:96::/112
+  kubeOvnCNI:
+    requests:
+      cpu: "100m"
+      memory: "100Mi"
+    limits:
+      cpu: "1"
+      memory:  "1Gi"
+  kubeOvnController:
+    requests:
+      cpu: "200m"
+      memory: "200Mi"
+    limits:
+      cpu: "1"
+      memory: "1Gi"
+  kubeOvnMonitor:
+    requests:
+      cpu: "200m"
+      memory: "200Mi"
+    limits:
+      cpu: "200m"
+      memory: "200Mi"
+  kubeOvnPinger:
+    requests:
+      cpu: "100m"
+      memory: "100Mi"
+    limits:
+      cpu: "200m"
+      memory: "400Mi"
+  kubeletConfig:
+    kubeletDir: /var/lib/kubelet
+  logConfig:
+    logDir: /var/log
+  masterNodesLabel: node-role.kubernetes.io/control-plane=true
+  networking:
+    defaultSubnet: ovn-default
+    defaultVPC: ovn-cluster
+    enableECMP: false
+    enableEIPSNAT: true
+    enableMetrics:  true
+    enableSSL: false
+    netStack: ipv4
+    networkType: geneve
+    nodeSubnet: join
+    ovnLeaderProbeInterval: 5
+    ovnNorthdNThreads: 1
+    ovnNorthdProbeInterval: 5000
+    ovnRemoteOpenflowInterval: 10
+    ovnRemoteProbeInterval: 10000
+    podNicType: veth-pair
+    probeInterval:  180000
+    tunnelType: vxlan
+    nodeLocalDNSIPS: ""
+    vlan:
+      providerName: provider
+      vlanId: 1
+      vlanName: ovn-vlan
+  openVSwitchDir: /var/lib/rancher/origin/openvswitch
+  ovnCentral:
+    requests:
+      cpu: 300m
+      memory: 200Mi
+    limits:
+      cpu: 3
+      memory: 4Gi
+  ovnDir: /etc/origin/ovn
+  ovsOVN:
+    limits:
+      cpu:  2
+      memory: 1000Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+  performance:
+    gcInterval: 360
+    inspectInterval: 20
+    ovsVSCtlConcurrency: 100
+```
+:::
+
+Users can fine tune the `Configuration` object by editing the `kubeovn-operator` addon as a yaml.
```

Comment:
We can emphasize that, generally, there's no need to adjust the default Configuration object, but if the user wants to fine-tune it, they can do so.

Reply:
sounds good. i will update along with the deletion instructions